process/subprocesses/_08_global_urban_heat_vulnerability_index.py

```diff
@@ -0,0 +1,1192 @@
+"""
+Generate global urban heat vulnerability index (GUHVI) through use of the Google Earth Engine API
+"""
+import sys
+import time
+
+# Import Earth Engine and Google Earth Engine Map libraries
+import ee
+import geemap
+import psycopg2
+import geopandas as gpd
+
+# import getpass
+from script_running_log import script_running_log
+from sqlalchemy import text
+
+import ghsci
+
+
+def fetch_guhvi_data_as_gdf(r: ghsci.Region) -> tuple:
```

Comment:
check for repeated definition of functions across different scripts --- use r.get_gdf() if it does the same thing